server/server.go

```diff
@@ -642,3 +642,22 @@ func (s *Server) GetMemberLeaderPriority(id uint64) (int, error) {
 func (s *Server) SetLogLevel(level string) {
 	s.cfg.Log.Level = level
 }
+
+var healthURL = "/pd/ping"
+
+// CheckHealth checks if members are health
+func (s *Server) CheckHealth(members []*pdpb.Member) map[uint64]*pdpb.Member {
+	unhealthMembers := make(map[uint64]*pdpb.Member)
+	for _, member := range members {
+		for _, cURL := range member.ClientUrls {
+			resp, err := DialClient.Get(fmt.Sprintf("%s%s", cURL, healthURL))
+			if err != nil || resp.StatusCode != http.StatusOK {
+				unhealthMembers[member.GetMemberId()] = member
```

Comment:
so as long as one client urls disconnected, we think this member is unhealthy? if so, no need iterate rest, just break.